var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -23,6 +23,7 @@ class Fontconfig(AutotoolsPackage):
     depends_on('pkgconfig', type='build')
     depends_on('font-util')
     depends_on('uuid', when='@2.13.1:')
+    depends_on('python', type='build')
```

Comment:
From the commit history python is only needed in `2.13.93` so this needs a `when` restriction. It also requires Python 3+.
```suggestion
    depends_on('python@3:', type='build', when='@2.13.93:')
```